src/index.ts

```diff
@@ -244,6 +254,63 @@ const tracker: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/*
+ * A plugin for add widget with detial of veriable.
+ */
+const details: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/debugger:details',
+  autoStart: true,
+  requires: [IDebugger],
+  activate: (app: JupyterFrontEnd, service: IDebugger) => {
+    const { commands, shell } = app;
+    const detailsTracker = new WidgetTracker<MainAreaWidget<VariableDetails>>({
```

Comment:
Should `detailsTracker` be called `tracker` if that's the only tracker in this plugin?